### PR TITLE
Created Dashboard.html

### PR DESCRIPTION
**Pull request by Vanoo123**
In Dashboard.html Created Buttons (Dashboard, Settings and Logout) styled it. in child 2 section created 2 Title and 3 BOX about day, week and month, also created one Table and styled it to.

### CODE REVIEW

File: dashboard.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+        <section class="child2">
+            <div class="dashboard_wrapper">
+                <div class="dashboard_title">
+                    <h1>Dashboard</h1>
+                </div>
+                <div class="view_title">
+                    <h3>Views</h3>
+                </div>
+                    <div class="three_box">
```
Comment (Giviko31) [inline dashboard.html:43]: good job my friend

File: dashboard.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                    </div>
+                <div class="page_box">
+                    <h3>Pages</h3>
+                </div>
+                <div class="table_box">
+                    <table>
+                        <tbody>
+                            <tr>
+                                <td>/ <span>500</span></td>
+                            </tr>
```
Comment (Giviko31) [inline dashboard.html:65]: nice one

File: styles/style.css [hunk 3]
```diff
@@ -38,4 +38,146 @@
...
+
+.page_box {
+    margin-top: 20px;
+}
+
+.table_box {
+    margin-top: 20px;
+}
+
+table {
```
Comment (AlexZviadadze) [inline styles/style.css:230]: Another  time  we can  create more beautiful tables  at this time is nice bro

**Review by Giviko31 — COMMENTED** [review]
..

File: dashboard.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
```
Comment (AlexZviadadze) [inline dashboard.html:1]: Nice  Job  u are good

**Review by AlexZviadadze — APPROVED** [review]

**Pull request merged**